src/dialogs.ycp

```diff
@@ -77,20 +77,20 @@ term EditDialogContents()
 					    `VBox(
 						  // combo box label: enable FCoE (yes/no)
 						  `ComboBox( `id(`fcoe), _("&FCoE Enable"),
-							     [`item(`id("yes"), "yes" ),
-							      `item(`id("no"), "no", true ) ]
+							     [`item(`id("yes"), _("Yes") ),
```

Comment:
Ooooooh, it's YCP! :zap: :grin: Use YCP equivalent of the above code.

BTW does it actually make sense to fix translations for SLE11-SP3? I would doubt even about SP4...